.github/workflows/handle_updates.yml

```diff
@@ -9,48 +9,38 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       # Check out and set up redox_lib_gen
-      - name: Checkout
+      - name: Checkout lib generator
         uses: actions/checkout@v3
         with:
           path: redox_lib_gen
 
       - name: Install Python and Poetry
-        uses: ITProKyle/action-setup-python@v0.3.0
+        uses: mmabey/action-setup-python@v0.3.1
```

Comment:
https://github.com/actions/setup-python

Would prefer the official setup-python action

Reply:
Thanks for taking a look Pen, I really appreciate it.

The official setup-python action is fine, but since Poetry also needs to be set up with the same version of Python, I was happy when I found an action that did both in one go. Once GitHub dropped legacy support for Node v12, the `ITProKyle/action-setup-python` action stopped working completely, which is what prompted a lot of the work in this PR.

The code of [my version of the action](https://github.com/mmabey/action-setup-python/blob/master/action.yml) does exactly 5 things:
1. Uses the official setup-python action
2. Sets up Poetry
3. Uses a cache for the virtual environment
4. Checks that the cache is still healthy
5. Installs the Poetry virtual environment and the packages as specified in the lock file

If you feel strongly about only using the official setup-python action, I can find a workaround. But I think that having those steps in their own action makes sense. And the only reason I started maintaining my own version of that was because the original author wasn't merging the Dependabot PRs.

Hopefully that context is helpful. Please let me know your thoughts!